.NET/Microsoft.Recognizers.Definitions.Common/Japanese/NumbersDefinitions.cs

```diff
@@ -51,6 +51,7 @@ public static class NumbersDefinitions
       public static readonly Dictionary<char, double> ZeroToNineMap = new Dictionary<char, double>
         {
             { '零', 0 },
+            { '〇', 0 },

```

Comment:
Is this still necessary as now half-width 0 will be caught in the regular digits regex?

Reply:
It's for Kanji.